src/main/java/jdwp/JDWPReferenceType.java

```diff
@@ -218,7 +218,8 @@ static class SourceFile implements Command  {
             static final int COMMAND = 7;
 
             public void reply(GDBControl gc, PacketStream answer, PacketStream command) {
-                ReferenceTypeImpl type = command.readReferenceType();
+                long id = command.readObjectRef();
```

Comment:
Why this separation of code?

Reply:
There's few more of these same changes in this file....can you revert all of them to not be separated?